libphal/phal_exception.H

```diff
@@ -24,6 +24,7 @@ enum ERR_TYPE {
 	SBE_CMD_FAILED,
 	SBE_CMD_TIMEOUT,
 	SBE_FFDC_NO_DATA,
+	SBE_BOOT_SEEPROMS_BAD,
```

Comment:
for internal consumption so existing one is ok.

Reply:
IMO, it is not for internal purposes because it is exposed in the exception header so the libphal user will do some action based on the error type. In the future, we may hit cases where we need to tell whether primary or secondary or both seeprom is bad so, My suggestion would be, let's add based on the current behavior.